src/picosvg/svg_types.py

```diff
@@ -79,25 +80,34 @@ def _visible(fill, opacity):
         )
 
     def bounding_box(self) -> Rect:
-        x1, y1, x2, y2 = svg_pathops.bounding_box(self)
+        x1, y1, x2, y2 = svg_pathops.bounding_box(self.as_cmd_seq())
         return Rect(x1, y1, x2 - x1, y2 - y1)
 
-    def apply_transform(self, transform: Affine2D):
-        return svg_pathops.transform(self, transform)
+    def apply_transform(self, transform: Affine2D) -> "SVGPath":
+        cmds = svg_pathops.transform(self.as_cmd_seq(), transform)
```

Comment:
this is where all the non-path related attributes get lost (https://github.com/googlefonts/picosvg/issues/32).
Before returning a new SVGPath constructed only from `cmds`, we should copy over to the new SVGPath all the rest of the attributes (excluding `transform` in this case).
Maybe we can define an _instance_ method called, I don't know, `replace_commands` (for want of better name) that takes self and the new commands and extra kwargs (to override specific attributes), and returns a new SVGPath with the new commands and replaced attributes.

Reply:
+1, but will send a separate PR